cfpb/consumerfinance.gov
# Don't show any cards for non-live portal pages

There was some logic on the ask landing page that would attempt to recover and show a reasonable card if a portal page was in draft. This was leading to duplicated cards when adding draft portal pages.

Testing:
 - Pull
 - Make sure you have a recent-ish db dump
 - open http://localhost:8000/ask-cfpb/. There should be no duplicated cards.
